packages/core/content-type-builder/admin/src/pages/ListView/index.js

```diff
@@ -224,7 +224,7 @@ const ListView = () => {
                 </Button>
                 <Button
                   startIcon={<CheckIcon />}
-                  onClick={submitData}
+                  onClick={() => submitData()}
```

Comment:
Why do you need to create another function?

Reply:
We don't want submiData to receive the event param